docs/RELEASE_NOTES.md

```diff
@@ -83,7 +83,8 @@ Based on work from Larry Person, there is a large refactor of "service-vendors"
 
 #### Database upgrades
 * This is a major release and includes a schema change.
-* For small instances simply leave/set SUPPRESS_MIGRATIONS="" or for [AWS Lambda, see the db migration instructions](https://moveonorg.github.io/Spoke/#/HOWTO_DEPLOYING_AWS_LAMBDA?id=migrating-the-database). 
+* For small instances simply leave/set SUPPRESS_MIGRATIONS="" or for [AWS Lambda, see the db migration instructions](https://moveonorg.github.io/Spoke/#/HOWTO_DEPLOYING_AWS_LAMBDA?id=migrating-the-database).
+* If you are running Spoke on Heroku and updating to version 11, either remove SUPPRESS_MIGRATIONS="" if it is set in config, *or* set SUPPRESS_MIGRATIONS=false
```

Comment:
I believe this same issue would occur in releases after Spoke 11 as well. Can we say "version 11 or later" instead of "version 11"?

```suggestion
* If you are running Spoke on Heroku and updating to version 11 or later, either remove SUPPRESS_MIGRATIONS="" if it is set in config, *or* set SUPPRESS_MIGRATIONS=false
```

Reply:
The same issue would occur on every version before 11 as well :) I just didn't update the previous entries to reflect that.